app/controllers/admin/volunteers.py

```diff
@@ -37,8 +38,9 @@ def trackVolunteersPage(eventID):
 
     trainedParticipantsList = trainedParticipants(program, g.current_term)
     eventParticipants = getEventParticipants(event)
+    programManagerResult = isProgramManagerForEvent(g.current_user, event)
```

Comment:
it's a good idea to name boolean variables so that they make expressions clearer. In this case, like `isProgramManager` or `canManageProgram`